core/src/main/scala/sbtorgpolicies/github/GitHubOps.scala

```diff
@@ -79,7 +79,7 @@ class GitHubOps[F[_]: ConcurrentEffect: Timer](
           path         <- EitherT.right(Sync[F].delay(file.getAbsolutePath()))
           content      <- EitherT(Sync[F].delay(fileReader.getFileContent(path)))
           relativePath <- EitherT.right(relativePath(file))
-        } yield (content, relativePath)
+        } yield (relativePath, content)
```

Comment:
my bad :/

Reply:
Haha, no worries, the real error is to have the `(String, String)` as response